react/components/EXPERIMENTAL_Table/hooks/useDensity.ts

```diff
@@ -0,0 +1,17 @@
+import { useState, useMemo } from 'react'
+import { getRowHeight } from '../util'
+
+const useDensity = (density: Density) => {
+  const [selectedDensity, setSelectedDensity] = useState<Density>(density)
+  const rowHeight = useMemo(() => getRowHeight(selectedDensity), [
```

Comment:
I don't think this memo is necessary, since the `getRowHeight` function isn't CPU-intensive.

be careful to not overuse the `useMemo` hook, since we are "paying" with a higher memory usage, for simple functions or functions that doesn't affect the rendering of the components we can call the function directly.

Reply:
solved